student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] As required for the project, kindly make sure the `initialize` constructor method extends parent's constructor by adding `@classroom` and a `parameter` for it. This means  constructor method should also include all parameters of a `Person` + `classroom`. Make sure you correct this.
```ruby
   def initialize(age, classroom, name = "Unknown", parent_permission = true)
   ...
```